mlflow/tracking/_tracking_service/client.py

```diff
@@ -486,8 +485,7 @@ def create_experiment(self, name, artifact_location=None, tags=None):
         """Create an experiment.
 
         Args:
-            name: The experiment name, which must be a unique string
-                with maximum length of 500 characters.
```

Comment:
https://github.com/mlflow/mlflow/blob/796cbcf31121496d6fe4f5111990fe91fa5be13b/mlflow/store/tracking/abstract_store.py#L109 also needs to be updated